event_attribution_reporting.md

```diff
@@ -0,0 +1,136 @@
+
+Event sources will only be able to generate a single attribution report, compared to the three reports for navigation sources. Alongside the 1 bit of data, this further restricts the total amount of trigger side data which can be associated with a single `attributionsourceeventid`. See [privacy considerations](#privacy-considerations) for why this is necessary.
+
+
+### Coarse Attribution Expiry
+
+Event sources will have their `attributionexpiry` rounded to the nearest whole day.
+
+The `attributionexpiry` of a source can be used to associate attributionsourceeventids with a coarse trigger side timestamp. Because event sources are easier to register en masse than navigation sources, it would allow a reporting origin to do this with a finer granularity.
+
+
+### Noise on the attribution triggering event
+
+Attribution sources registered without a navigation will receive additional noise on whether attribution was triggered for the source. This is similar to the speculative noise discussed in the [Click Through Explainer](https://github.com/WICG/conversion-measurement-api#speculative-adding-noise-to-the-attribution-of-the-source-itself). 
+
+When an attribution source is registered, the browser will perform one of the following steps given noise `p`:
+* With probability `1 - p`, the browser logs the source as normal
+* With probability `p/2`, the browser ignores any future attribution for this source
+* With probability `p/2`, the browser immediately triggers attribution on the source with a random `event-source-trigger-data`. This report will still be sent at the end of the reporting window.
+
+These steps will not be performed for navigation attribution sources.
+
+See the [privacy considerations section](#privacy-considerations) for the rationale behind adding this noise.
+
+
+### Controlling which attribution source to trigger
+
+In the Click Through Explainer, at triggering time, the browser looks up all matching sources in storage. A [credit](https://github.com/WICG/conversion-measurement-api#multiple-sources-for-the-same-trigger-multi-touch) of 100 is assigned to the most recent matching source in storage, and a credit of 0 is assigned to the rest. Reports are sent for each of these. We propose changing this for both click sources and event sources.
+
+To provide additional utility in prioritizing the distribution of credit between different classes of sources, the browser can associate a priority with each attribution source. This priority will be defined in a new attribute on attribution source tags, `attributionsourcepriority`, which holds a 64 bit integer.
+
+When a trigger redirect is received, the browser will find the matching source with highest `attributionsourcepriority` value and generate a report. The other sources will not generate reports.
+
+If reports were sent for these sources, it would be possible for `reportingorigin` to use these other reports to remove the noise on the attribution triggering event. 
```

Comment:
```suggestion
If reports were sent for these sources, it would be possible for `reportingorigin` to use these other reports to remove the noise on the attribution triggering event, unless we introduced a mechanism to noise the sources dependently on others. 
```